staging/src/k8s.io/apiserver/pkg/audit/policy/reader_test.go

```diff
@@ -127,3 +135,24 @@ func TestParserV1beta1(t *testing.T) {
 		t.Errorf("Unexpected policy! Diff:\n%s", diff.ObjectDiff(policy, expectedPolicy))
 	}
 }
+
+func TestPolicyCntCheck(t *testing.T) {
+
+	test := func(policyFileName string) {
```

Comment:
Try this approach instead of creating a function: https://golang.org/doc/code.html#Testing

Also, you can use the following patter to run test cases:

```
for _, tc := testCases {
  t.Run(tc.desc, func() {
    ...
  })
}
```

Reply:
I'll study it